stack-packs/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "@lighthouse/stack-packs",
+  "version": "1.0.0",
+  "description": "Lighthouse",
+  "main": "./index.js",
+  "author": "The Chromium Authors",
+  "typings": "./index.d.ts"
```

```suggestion
  "types": "./index.d.ts"
```
come on, what year is it :)

---

i always use typings 🤷‍♂️